src/main/java/uk/gov/hmcts/darts/arm/service/DetsToArmProcessor.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.darts.arm.service;
+
+
+public interface DetsToArmProcessor {
```

Comment:
this is the Push processor, I'd suggest we mention that in the processor name. Because we are introducing also another Dets to Arm process which is for the pull

Reply:
@mario-paniccia  I'll rename it my PR as I am populating this.